robotbase/src/main/java/frc/robot/subsystems/CommandSwerveDrivetrain.java

```diff
@@ -118,4 +122,10 @@ public void accept(ChassisSpeeds speeds) {
       }
     };
   }
+
+  public double getSpeakerLockRotation() {
+    double tx = Robot.shooterLimelight.getTX();
```

Comment:
You may want to check if there is a valid target, or make sure the tx returns zero when it does not see a target